sonic-slave-stretch/Dockerfile.j2

```diff
@@ -239,6 +239,8 @@ RUN apt-get update && apt-get install -y \
 {%- endif %}
 # For lockfile
         procmail \
+# For gmock
+        libgmock-dev -t stretch-backports \
```

Comment:
>stretch-backports [](http://example.com/codeflow?start=24&length=17)

This option will impact all the packages in the same command line.

Reply:
Thanks, we should add a new command to install the backport package.